core/database/foxx/api/topic_router.js

```diff
@@ -154,7 +226,26 @@ router
             }
 
             res.send(result);
+            logger.logRequestSuccess({
+                client: client?._id,
+                correlationId: req.headers["x-correlation-id"],
+                httpVerb: "GET",
+                routePath: basePath + "/search",
+                status: "Success",
+                description: "Search topics",
+                extra: result,
```

Comment:
Do you see any problems with result here? I want you to tell me what is wrong.

<img width="1071" height="33" alt="image" src="https://github.com/user-attachments/assets/5b4b2295-cf0f-49a0-af91-af8b36ce5561" />